### PR TITLE
[NOMERGE] Removed licenses settings.

### PR DESCRIPTION
**Pull request by jeffmay**
@chrissalij-r @patrick-bohan @slustbader 

This seemed to work when I published it locally. Fingers crossed. Sorry I had to spam the release version up to 1.1.5. Now I know how to test without doing that tho.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -39,6 +37,3 @@ sources in(Compile, doc) := Seq.empty
 
 // disable publishing empty ScalaDocs
 publishArtifact in (Compile, packageDoc) := false
-
-licenses += ("Apache-2.0", url("http://opensource.org/licenses/apache-2.0"))
```
Comment (jeffmay) [inline build.sbt]: This was the fix.

**Comment by jeffmay** [comment]
Nope. Nvm. Didn't work.

File: build.sbt (hunk 1 above)
Comment (slustbader) [inline build.sbt]: Was the plugin picking up "source" from opensource.org?

Comment (jeffmay) [inline build.sbt]: This is actually a plain SBT setting. I thought it was specific to the sbt-bintray plugin.

**Pull request closed without merging**